feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreen.kt

```diff
@@ -9,10 +14,63 @@ import com.epmedu.animeal.navigation.navigator.LocalNavigator
 @Composable
 fun AboutScreen() {
     val navigator = LocalNavigator.currentOrThrow
+    val context = LocalContext.current
+    val facebookWebIntent = remember {
```

Comment:
What do you think about extracting all these intents to extension functions like `Context.openFacebook()`?